docs/core/tutorials/unloadability-howto.md

```diff
@@ -0,0 +1,359 @@
+
+Please note that if you debug using LLDB on Unix, the SOS commands in the examples below don't have the `!` in front of them.
+```
+!dumpheap -type LoaderAllocator
+```
+It will dump all objects with type name containing `LoaderAllocator` string that are in the GC heap. Here is an example:
+```
+         Address               MT     Size
+000002b78000ce40 00007ffadc93a288       48     
+000002b78000ceb0 00007ffadc93a218       24     
+
+Statistics:
+              MT    Count    TotalSize Class Name
+00007ffadc93a218        1           24 System.Reflection.LoaderAllocatorScout
+00007ffadc93a288        1           48 System.Reflection.LoaderAllocator
+Total 2 objects
+```
+In the "Statistics:" part below, check the `MT` (`MethodTable`) belonging to the `System.Reflection.LoaderAllocator` which is the object we care about. Then in the list at the beginning, find the entry with `MT` matching that one and get the address of the object itself. In our case, it is "000002b78000ce40"
+
+Now that we know the address of the `LoaderAllocator` object, we can use another command to find its GC roots
+```
+!gcroot -all 0x000002b78000ce40 
+```
+This will dump the chain of object references that lead to the `LoaderAllocator` instance. The list starts with the root, which is the entity that keeps our `LoaderAllocator` alive and thus is the core of the problem we are debugging. The root can be a stack slot, a processor register, a GC handle or a static variable.
+
+Here is an example of the output of the `gcroot` command:
+```
+Thread 4ac:
+    000000cf9499dd20 00007ffa7d0236bc example.Program.Main(System.String[]) [E:\unloadability\example\Program.cs @ 70]
+        rbp-20: 000000cf9499dd90
+            ->  000002b78000d328 System.Reflection.RuntimeMethodInfo
+            ->  000002b78000d1f8 System.RuntimeType+RuntimeTypeCache
+            ->  000002b78000d1d0 System.RuntimeType
+            ->  000002b78000ce40 System.Reflection.LoaderAllocator
+
+HandleTable:
+    000002b7f8a81198 (strong handle)
+    -> 000002b78000d948 test.Test
+    -> 000002b78000ce40 System.Reflection.LoaderAllocator
+
+    000002b7f8a815f8 (pinned handle)
+    -> 000002b790001038 System.Object[]
+    -> 000002b78000d390 example.TestInfo
+    -> 000002b78000d328 System.Reflection.RuntimeMethodInfo
+    -> 000002b78000d1f8 System.RuntimeType+RuntimeTypeCache
+    -> 000002b78000d1d0 System.RuntimeType
+    -> 000002b78000ce40 System.Reflection.LoaderAllocator
+
+Found 3 roots.
+```
+Now you need to figure out where is the root located so that you can fix it. The easiest case is when the root is a stack slot or a processor register. In that case, the `gcroot` shows you the name of the function whose frame contains the root and the thread executing that function. The difficult case is when the root is a static variable or a GC handle. 
```

Comment:
```suggestion
Now you need to figure out where the root is located so that you can fix it. The easiest case is when the root is a stack slot or a processor register. In that case, the `gcroot` shows you the name of the function whose frame contains the root and the thread executing that function. The difficult case is when the root is a static variable or a GC handle. 
```